config/pumi_config.h.in

```diff
@@ -0,0 +1 @@
+#cmakedefine MFEM_USE_SIMMETRIX
```

Comment:
Why the separate file? Why can't this be in `config/cmake/config.hpp.in`?

Reply:
Sure, I'll see if we can use `config.hpp.in`.

Reply:
fixed https://github.com/mfem/mfem/pull/237/commits/6601d3757b278a069dc6afb9bc72463f52d986ed